spinoso-time/src/time/chrono/build.rs

```diff
@@ -50,6 +51,38 @@ impl Time {
             offset,
         }
     }
+
+    /// Creates a new `Time` object from the `seconds` and `sub_seconds_nanos`
+    /// since the Unix EPOCH with a local offset.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use spinoso_time::Time;
+    /// let epoch = Time::at(0, 0);
+    /// let epoch_plus_1_nano = Time::at(0, 1);
+    /// ```
+    #[inline]
+    #[must_use]
+    pub fn at(seconds: i64, sub_second_nanos: i64) -> Self {
+        let offset = Offset::Local;
+
+        let timestamp = seconds + (sub_second_nanos / i64::from(NANOS_IN_SECOND));
```

Comment:
let's used a checked addition here like my suggestion above in the trampoline so we don't accidentally wrap around. (in debug mode this would be a panic, which you can test by doing a `Time.at(1e60, 1e60)`.